pkg/cmd/server/api/validation/validation_test.go

```diff
@@ -206,3 +211,41 @@ MCUXgckTpKCuGwbJk7424Nb8bLzf3kllAiA5mUBgjfr/WtFSJdWcPQ4Zt9KTMNKD
 EUO0ukpTwEIl6wIhAMbGqZK3zAAFdq8DD2jPx+UJXnh0rnOkZBzDtJ6/iN69AiEA
 1Aq8MJgTaYsDQWyU/hDq5YkDJc9e9DSCvUIzqxQWMQE=
 -----END RSA PRIVATE KEY-----`)
+
+func TestLDAPSyncConfigFixtures(t *testing.T) {
+	fixtures := []string{}
+
+	// build a list of common configurations for all schemas
+	schemas := []string{"rfc2307", "ad", "augmented-ad"}
+	for _, schema := range schemas {
```

Comment:
Or I guess we could change the way we export groups and validate that the other files are `GroupLists`... Unless I can decode a file into a slice of groups with the codec?